schemas/compat-data-schema.md

```diff
@@ -249,6 +249,26 @@ Examples:
 }
 ```
 
+### Initial versions
+
+The following table indicates initial versions for browsers in BCD. These are the earliest possible version numbers allowed to be used. These are not naturally "1" or "1.0" as explained in the notes column.
+
+| Browser          | Initial version | Notes                                                                                                                      |
+| ---------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Chrome           | 1               |                                                                                                                            |
+| Chrome Android   | 18              | First stable release started at 18. No Chrome Android 17 or earlier was ever released.                                     |
+| Edge             | 12              | EdgeHTML, continuation of IE 11 (another initial version is Edge 79 when Edge switched to Chromium).                       |
+| Firefox          | 1               |                                                                                                                            |
+| Firefox Android  | 4               | First stable release started at 4. Earlier non-Android mobile versions are ignored.                                        |
+| IE               | 1               |                                                                                                                            |
+| Node.js          | 0.1.100         | Subject to change to `0.10.0` which is the first stable version (LTS status was retrospectively applied). See issue #6861. |
```

Comment:
suggestion: wordsmithing to avoid the whole retrospective bit

(Technically, Node.js awards LTS status after an even version has shipped and has proven to be reliable after a few months—it doesn't get it immediately. It's a bit of detail I didn't want to have to explain in the doc.)

Also, I removed the backticks on the versions here because they weren't used elsewhere in the table, but I wouldn't mind seeing them reintroduced throughout.

```suggestion
| Node.js          | 0.1.100         | Subject to change to 0.10.0. This project selected 0.10.0 as the first release primarily because the `0.10`-series releases was the first have LTS status applied. See issue #6861. |
```